metals/src/main/scala/scala/meta/internal/metals/Embedded.scala

```diff
@@ -119,9 +119,12 @@ object Embedded {
       dep: Dependency,
       scalaVersion: String
   ): Fetch = {
+
     val resolutionParams = ResolutionParams
       .create()
-      .forceVersions(
+
+    if (!ScalaVersions.isScala3Version(scalaVersion))
```

Comment:
Removed the redundant method for create Scala 3 presentation compiler and just added this single condtion. Everything else is the same.